### PR TITLE
{bio}[foss/2019b] medaka v1.4.3 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node2643.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/cf6f87d5a475cfea6bb339eafd71769a for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3120.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/051d6620a9bd6ee8af4ec8a21c9bc060 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3509.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/c2f472cbd299dc0f3751cda6ad88f359 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (3 easyconfigs in total)
bear-pg0206u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/039bbe357698a18c82d5553469008b10 for a full test report.

File: easybuild/easyconfigs/m/medaka/medaka-1.4.3-foss-2019b-Python-3.7.4.eb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+homepage = 'https://github.com/nanoporetech/medaka'
+description = "medaka is a tool to create a consensus sequence from nanopore sequencing data."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+
+builddependencies = [('Autotools', '20180311')]
+
+dependencies = [
+    ('Python', '3.7.4'),  # includes cffi, numpy
+    # TensorFlow 2.2.2 required by medaka 1.2.6, see requirements.txt
```
Comment (branfosj) [inline easybuild/easyconfigs/m/medaka/medaka-1.4.3-foss-2019b-Python-3.7.4.eb:21]:
```suggestion
    ('Python', '3.7.4'),  # includes cffi
    # TensorFlow 2.2.2 required by medaka 1.4.3, see requirements.txt
```

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**